src/libstd/process.rs

```diff
@@ -170,17 +170,32 @@ pub struct Child {
     handle: imp::Process,
 
     /// The handle for writing to the child's standard input (stdin), if it has
-    /// been captured.
+    /// been captured. You might find it helpful to do
+    /// ```
+    /// let stdin = child.stdin.take().unwrap();
+    /// ```
+    /// to avoid partially moving the `child` and thus block yourself from calling
+    /// functions on `child` while using `stdin`.
```

Comment:
This may be clearer?

```suggestion
    /// been captured. To avoid partially moving
    /// the `child` and thus blocking yourself from calling
    /// functions on `child` while using `stdin`,
    /// you might find it helpful:
    /// ```
    /// let stdin = child.stdin.take().unwrap();
    /// ```
```